dd-java-agent/instrumentation/jetty-8/src/main/java/datadog/trace/instrumentation/jetty8/JettyHandlerAdvice.java

```diff
@@ -49,8 +49,8 @@ public static Scope startSpan(
     final Span span = scope.span();
     Tags.COMPONENT.set(span, "jetty-handler");
     Tags.HTTP_METHOD.set(span, req.getMethod());
-    Tags.HTTP_URL.set(span, req.getRequestURL().toString());
     span.setTag(DDTags.RESOURCE_NAME, resourceName);
+    Tags.HTTP_URL.set(span, req.getRequestURL().toString());
```

Comment:
Url needed to be set after resource name tag to prevent overwriting the decorator's change.